cloud/main.js

```diff
@@ -14,7 +14,7 @@ Parse.Cloud.define("deleteFile", function(request, response) {
   } else {
 	  Parse.Cloud.httpRequest({
 		method: 'DELETE',
-		url: 'https://parse-parse4cn1.rhcloud.com/parse/files/' + filename,
```

Comment:
parse4cn1 runs against openshift by default, not back4app so please revert this change otherwise tests will fail.